kSuite/kSuitePro/src/main/kotlin/com/infomaniak/core/ksuite/ksuitepro/views/components/ProOfferContent.kt

```diff
@@ -0,0 +1,222 @@
+import com.infomaniak.core.compose.basics.Typography
+import com.infomaniak.core.compose.margin.Margin
+import com.infomaniak.core.ksuite.data.KSuite
+import com.infomaniak.core.ksuite.ksuitepro.R
+import com.infomaniak.core.ksuite.ksuitepro.data.ProFeature
+import com.infomaniak.core.R as RCore
+
+@Composable
+fun ProOfferContent(
+    kSuite: KSuite,
+    isAdmin: Boolean,
+    onClick: () -> Unit,
+) {
+
+    val title = computeTitle(kSuite)
+    val description = computeDescription(kSuite)
+    val features = computeFeatures(kSuite)
+
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+            .background(colorResource(R.color.kSuiteContentBackground)),
+        horizontalAlignment = Alignment.CenterHorizontally,
+    ) {
+
+        val paddedModifier = Modifier.padding(horizontal = Margin.Large)
+
+        Image(
+            imageVector = ImageVector.vectorResource(R.drawable.illu_ksuite_header),
+            contentScale = ContentScale.FillWidth,
+            contentDescription = null,
+            modifier = Modifier.fillMaxWidth(),
+        )
+
+        Spacer(Modifier.height(Margin.Large))
+        Text(
+            modifier = paddedModifier,
+            text = title,
+            textAlign = TextAlign.Center,
+            style = Typography.h2,
+            color = colorResource(R.color.kSuitePrimaryText),
+        )
+
+        Spacer(Modifier.height(Margin.Medium))
+        Text(
+            modifier = paddedModifier,
+            text = description,
+            style = Typography.bodyRegular,
+            color = colorResource(R.color.kSuiteSecondaryText),
+        )
+
+        Spacer(Modifier.height(Margin.Medium))
+        ProFeatures(paddedModifier, features)
+
+        Spacer(Modifier.height(Margin.Large))
+        Text(
+            modifier = paddedModifier,
+            text = stringResource(if (isAdmin) R.string.kSuiteUpgradeDetails else R.string.kSuiteUpgradeDetailsContactAdmin),
+            style = Typography.bodyRegular,
+            color = colorResource(R.color.kSuiteSecondaryText),
+        )
+
+        Spacer(Modifier.height(Margin.Huge))
+        Button(
+            modifier = Modifier
+                .padding(horizontal = Margin.Medium)
+                .fillMaxWidth()
+                .height(Dimens.buttonHeight),
+            colors = ButtonDefaults.buttonColors(
+                containerColor = colorResource(R.color.kSuiteButtonBackground),
+                contentColor = colorResource(R.color.kSuitePrimaryText),
+            ),
+            shape = RoundedCornerShape(Dimens.largeCornerRadius),
+            onClick = onClick,
+        ) {
+            Text(text = stringResource(RCore.string.buttonClose), style = Typography.bodyMedium)
+        }
+
+        Spacer(Modifier.height(Margin.Large))
+    }
+}
+
+@Composable
+private fun ProFeatures(
+    modifier: Modifier,
+    features: List<ProFeature>,
+) {
+    val featureModifier = Modifier.fillMaxWidth()
```

Comment:
I'm not sure this is really useful. 
You should probably recheck all the alignement for the landscape mode, i'm not sure they're working